sources/personio/README.md

```diff
@@ -52,20 +64,22 @@ any of the other [destinations](https://dlthub.com/docs/dlt-ecosystem/destinatio
    pip install -r requirements.txt
    ```
 
-2. Now the pipeline can be run by using the command:
+1. Now the pipeline can be run by using the command:
 
    ```bash
    python3 personio_pipeline.py
    ```
 
-3. To make sure that everything is loaded as expected, use the command:
+1. To make sure that everything is loaded as expected, use the command:
 
    ```bash
    dlt pipeline personio_pipeline show
```

Comment:
`dlt pipeline <pipeline_name> show`

Reply:
![Screenshot 2023-12-07 at 14 17 11](https://github.com/dlt-hub/verified-sources/assets/98139823/21677e51-8dce-4625-97fb-3f641118ed3a)

Did that in the last comment. I wonder why it showed otherwise.